rexray/cli/cmds_volume.go

```diff
@@ -66,14 +66,13 @@ func (c *CLI) initVolumeCmds() {
 			}
 			if c.volumeID != "" || c.volumeName != "" {
 				for _, v := range vols {
-					if strings.ToLower(v.ID) == strings.ToLower(c.volumeID) ||
-						strings.ToLower(v.Name) == strings.ToLower(c.volumeName) {
+					if (c.volumeID != "" && strings.ToLower(v.ID) == strings.ToLower(c.volumeID)) ||
+						(c.volumeName != "" && strings.ToLower(v.Name) == strings.ToLower(c.volumeName)) {
```

Comment:
Yes, we were wondering if the change you requested was complicated or if the code I posted above was sufficient. My apologies, I should have been more clear in my line comment.
Thank you for your help.